baybe/kernels/basic.py

```diff
@@ -1,12 +1,14 @@
 """Collection of kernels."""
 
 from fractions import Fraction
-from typing import Union
+from typing import Optional, Union
 
 from attrs import define, field
+from attrs.converters import optional
 from attrs.validators import in_
 
 from baybe.kernels.base import Kernel
+from baybe.kernels.priors.base import Prior
 
 
 def _convert_fraction(value: Union[str, float, Fraction], /) -> float:
```

Comment:
should this be a utility in utils.numerical?

Reply:
Have no strong opinion, what is @AdrianSosic opinion?

Reply:
I've moved it to a new file `conversion.py` (similar to our `validation.py`) where we can maintain custom converters from now on 👍🏼